src/main/java/hudson/plugins/jira/versionparameter/VersionComparator.java

```diff
@@ -42,7 +42,10 @@ public int compare(Version rev1, Version rev2) {
     protected String getNumberVersion(String firstV) {
         String res = firstV;
         if (!firstV.matches("[0-9.]+") && firstV.contains("-")) {
```

Comment:
can you pls also remove number regexp matching here since it will never match if there is a '-' in the input string anyway?

Reply:
Indeed. I didn't want modify the existing condition, but you're right.

Adding commit is ok ? Or do you want me to rebase?